tests/README.md

```diff
@@ -0,0 +1,104 @@
+Jetty Testing Design
+==
+
+The goal of testing in the context of the vanilla runtime is to provide an extensible framework with two modes, one for local testing and validation of an application and one for remote testing which deploys applications as services into the GCE.  Tests are not a part of the normal build process, they are a separate process with both local and remote components.  Additionally tests are conceptually disconnected from the actual build of the jetty-runtime project and should be considered as something that can pulled into their own github repository with minimal changes required to the core jetty-runtime build or project.
+
+
+Execution Examples:
+===
+
+Below are a series of commands and their expected behaviors.
+
+
+Standard Execution
+====
+
+```
+> mvn install
```

Comment:
Since the file is now under /tests, we need to clarify that this refers to running the build for the runtime from ../. Also, I think we need to clarify that running 'mvn install' under /tests, will not actually run any tests.